### PR TITLE
(1257) Report Channel of delivery code

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR

- Allow Channel of delivery code to be imported from CSV and reported in the CSV

## Screenshots of UI changes

N/A

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by CristinaRO** [comment]
[I can report Channel of delivery code](https://trello.com/c/FVozLelR/1257-i-can-report-channel-of-delivery-code)

File: spec/services/activities/import_from_csv_spec.rb [hunk 1]
```diff
@@ -162,53 +164,41 @@
...
-        },
+        invalid_activity_attributes,
       ]
 
       expect { subject.import(activities) }.to_not change { existing_activity }
 
       expect(subject.created.count).to eq(0)
       expect(subject.updated.count).to eq(0)
-
-      expect(subject.errors.count).to eq(1)
```
Comment (leeky) [inline spec/services/activities/import_from_csv_spec.rb:180]: As the `it` block starts with "has an error", we should probably leave the expectation to check that there is an error in, even if we're no longer checking for an exact error message.

File: spec/services/activities/import_from_csv_spec.rb [hunk 2]
```diff
@@ -162,53 +164,41 @@
...
```
Comment (leeky) [inline spec/services/activities/import_from_csv_spec.rb:206]: Same here.

**Review by leeky — APPROVED** [review]
Two comments about restoring some expectations that were removed, but otherwise LGTM! :shipit:

**Review by leeky — APPROVED** [review]

**Pull request merged**